src/Analyzers/Core/Analyzers/IDEDiagnosticIds.cs

```diff
@@ -152,6 +152,8 @@ internal static class IDEDiagnosticIds
         public const string RemoveRedundantEqualityDiagnosticId = "IDE0100";
 
         public const string RemoveUnnecessaryDiscardDesignationDiagnosticId = "IDE0110";
+        
+        public const string NamespaceSyncAnalyzerDiagnosticId = "IDE0085";
```

Comment:
Should be situated under IDE0083?

Reply:
I have the same question.

---
In reply to: [545566995](https://github.com/dotnet/roslyn/pull/49990#discussion_r545566995) [](ancestors = 545566995)